flycheck.el

```diff
@@ -1803,14 +1802,15 @@ CHECKER is a checker symbol.
 
 Pop up a help buffer with the documentation of CHECKER."
   (interactive
-   (let* ((checker (flycheck-checker-at-point))
+   (let* ((checker-at-pt (flycheck-checker-at-point))
```

Comment:
I think this binding is redundant.  It's only used at one place, so we can just as well inline it there, can't we?